### ISSUE TITLE
`--version` option

### ISSUE THREAD

**Issue by AlexWayfer** [cirruslabs/cirrus-cli#73]
Hello.

I think it'd be handful to have some `-v` or `--version` or `version` option/command to debug which one `cirrus-cli` you're using exactly.

Current output:

```
> cirrus help
Cirrus CLI

Usage:
  cirrus [command]

Available Commands:
  help        Help about any command
  run         Execute Cirrus CI tasks locally
  validate    Validate Cirrus CI configuration file

Flags:
  -h, --help   help for cirrus

Use "cirrus [command] --help" for more information about a command.
```

**Comment by fkorotkov** [comment]
Should be fixed via #65 🤔

**Comment by AlexWayfer** [comment]
> Should be fixed via #65

I've used `GO111MODULE=on go get github.com/cirruslabs/cirrus-cli/...` command from `INSTALL.md`, I guess it's using not `master` branch or something like this, so… what I should to do or wait for?

**Comment by edigaryev** [comment]
I've investigated this a bit and it seems that even if we'll retrieve [debug.BuildInfo](https://golang.org/pkg/runtime/debug/#BuildInfo) it'll be useless at best, since it's `Version` field [always returns `(devel)`](https://github.com/golang/go/issues/29228).

The other option is to hardcode the version in the code and update it each time we make a release, but it'd quite easy to mess that up. Besides that, showing that version might actually be a misnomer in cases where the user mistakenly compiles the CLI from a dirty state or with some modifications made by a mistake.

Perhaps we should rather demote the Go installation method to be the least desirable/for development purposes only and be done with it.

@fkorotkov what do you think?

**Comment by fkorotkov** [comment]
Oh. I re-read your comment and just noticed that you tried `debug.BuildInfo`. Don't know why I missed it. Are you possitive that it is always `(devel)`? I saw `(devel)` locally but I think with `install` it might get the sha. 🤔

**Comment by fkorotkov** [comment]
I saw [this line](https://github.com/golang/tools/blob/7db4ac6d26618953d34baaea10596dd8904562f5/internal/lsp/debug/info.1.12.go#L30) and though that it might work in case of `go get`

**Comment by edigaryev** [comment]
This was fixed in #77 and now that `0.7.0` covering these changes is released, `--version` flag works as intended:

```
# (cd && GO111MODULE=on go get github.com/cirruslabs/cirrus-cli/...)
go: downloading github.com/cirruslabs/cirrus-cli v0.7.0
go: found github.com/cirruslabs/cirrus-cli/... in github.com/cirruslabs/cirrus-cli v0.7.0
[...]
# cirrus --version
cirrus version v0.7.0
```
